### PR TITLE
Update documentation and JS examples after 140122 release

### PR DESCRIPTION
**Pull request by alexromanov**

### CODE REVIEW

File: docs/source/guides/store_and_hash_service_guide.md [hunk 1]
```diff
@@ -1,30 +1,32 @@
 # The StoreAndHash Service
 
-The StoreAndHash service provides an API for posting data to AWS S3 storage, getting the hash of the data and URL for further download.
+The StoreAndHash service provides an API for posting data to [AWS S3](https://aws.amazon.com/s3/) or [IPFS](https://ipfs.io/) storages, getting the HASH of the data and URL for further download.
```
Comment (mcsherrylabs) [inline docs/source/guides/store_and_hash_service_guide.md:3]: HASH -> hash (no need for capitals)

File: examples/js/README.md [hunk 2]
```diff
@@ -28,33 +28,66 @@ grpc_tools_node_protoc --proto_path=../../protos/cardano-metadata-service/protob
 2. Set your username and API Token in the src/metadata_service_client.js file:
 ```shell
 const credentials = new CommonMessages.CredentialsMessage()
-    .setClientId("your_psg_username")
-    .setApiToken("your_api_token");
+    .setClientId("CLIENT_ID")
```
Comment (mcsherrylabs) [inline examples/js/README.md:31]: Add in the comment that CLIENT_ID and API_TOKEN needs to be replaced (similarly to how it's done below)

**Review by mcsherrylabs — APPROVED** [review]
2 minor comments.... :+1:

File: docs/source/guides/store_and_hash_service_guide.md (hunk 1 above)
Comment (alexromanov) [reply docs/source/guides/store_and_hash_service_guide.md]: Fixed

File: examples/js/README.md (hunk 2 above)
Comment (alexromanov) [reply examples/js/README.md:31]: Fixed

**Pull request merged**